### DISCUSSION TITLE
runc seccomp_notify_respond

### DISCUSSION THREAD

**Discussion by tanberBro** [opencontainers/runc#3414] [Q&A] [answered]
when use runc ctl：
$runc
runc: symbol lookup error: runc: undefined symbol: seccomp_notify_respond

What is the reason and how to solve it?
system：centos7.9
I have executed this command: yum install -y libseccomp-devel

**Comment by kolyshkin** [comment, marked as answer]
Recent runc requires libseccomp 2.5.0. If your distro does not provide it, you can use a static build from the releases page.
